crypto.c

```diff
@@ -75,6 +75,38 @@ char *read_passphrase_twice(const char *prompt)
 	return pass;
 }
 
+char *read_password_from_file(const char *file_path) {
```

Comment:
We already have a helper for this - read_file_str() in tools-util.c

Reply:
Nice. I'll use that! Thanks for your time.

Reply:
I finally got the time to make the appropriate change.